cmd/up.go

```diff
@@ -42,6 +44,12 @@ func runUp(_ *cobra.Command, _ []string) error {
 		clientSuffix = "-arm64"
 	}
 
+	authFileErr := makeBasicAuthFiles()
+	if authFileErr != nil {
+		log.Println(fmt.Sprintf("Error creating basic auth files, %q", authFileErr))
```

Comment:
Can you do `errors.Wrap` instead and just return it? No need to print and return, it's printed one level up. This may result in the message being printed twice.